### DISCUSSION TITLE
Popper is evolving into Floating UI! 💥

### DISCUSSION THREAD

**Discussion by FezVrasta** [floating-ui/floating-ui#1425] [General]
You may have noticed the repository and the Popper organization names changed.

@atomiks worked hard on a new version of Popper that is more focused on composability, cross-platform compatibility, bundle size, and a more modern API.

The changes were big enough to deserve a major release, but we also wanted to rebrand the project to something easier to recognize and more meaningful, so welcome **Floating UI** and say goodbye to Popper!

But wait, what'll happen to Popper (`@popperjs/core@2.x`)? The library is going to be kept in low maintenance mode and a new library (`@floating-ui/popper`) will be developed in the near future to provide a backward compatibility layer between the old API and the new one.

In the upcoming days we'll update the website and move the projects all under this new monorepo.

For all the people who may be concerned about the future of our adorable mascotte, don't worry, the pop-corn guy will not go anywhere! 🍿

@atomiks and me are here to answer all your questions so feel free to join the conversation!

**Comment by pleunv** [comment]
Hi peeps! Absolutely blown away by the ease-of-use of @floating-ui and glad you're taking this direction with popper. Just popped it in a project to work around some of `react-select`'s positioning and clipping issues and it just works with little to no effort, even in IE11! Totally amazeballs.

**Reply by atomiks** [reply]
@pleunv you must have a lot of polyfills for IE11 then 😉  it relies on Promise, Array.prototype.includes, String.prototype.includes, Object.assign, and a bunch more. + you'd have to be compiling syntax in `node_modules` so it's kind of a miracle it works haha

**Reply by pleunv** [reply]
Exactly... Just for a couple more months 😄

**Comment by jvanoostveen** [comment]
I was wondering if there is a migration guide from Popper 2 to Floating UI.
An backwards compatibility layer would be welcome indeed, but I don't mind changing our code. But I'm having some trouble 'converting' the code because of changed concepts and such.
Thanks!

**Reply by FezVrasta** [reply]
Hey, unfortunately the compat layer is lagging behind, I'm particularly busy lately and I didn't have time to work on it yet, realistically I don't think I'll be able to work on it anytime soon.

A migration guide would indeed make sense.
